golem-test-framework/src/components/component_compilation_service/docker.rs

```diff
@@ -39,13 +39,6 @@ impl DockerComponentCompilationService {
     pub async fn new(
```

Comment:
unrelated cleanups here

Reply:
Why did you change this / what was the two kind of constructors used before?

Reply:
Just a leftover from another pr. They used to be used to set a default for a specific kind of env vars, but I got rid of it entirely as there is only a single implementation across both oss and cloud https://github.com/golemcloud/golem/pull/1390/files#diff-defcdcc5cbd9c47aee747607c42d64036dea791683cfa67c00448bb1b9aea0d5L51

Just forgot to delete these now redundant constructors